.github/actions/setup-rocm/action.yml

```diff
@@ -59,11 +59,6 @@ runs:
             echo "$msg"
             exit 1
         fi
-        if [[ $ngpu -eq 1 ]]; then
-            echo "Error: only 1 GPU detected, at least 2 GPUs are needed for distributed jobs"
-            echo "$msg"
-            exit 1
-        fi
```

Comment:
@saienduri How about we add an equivalent check in https://github.com/pytorch/pytorch/blob/main/.github/workflows/_rocm-test.yml which checks the `matrix.config` value to ensure that `distributed` jobs have 4GPUs visible? That was the main reason for introducing this check.